d2.config.js

```diff
@@ -1,11 +1,11 @@
 const config = {
     type: 'app',
-    name: 'app-management',
+    name: 'dhis-web-app-management',
```

Comment:
```suggestion
    name: 'app-management',
```

Reply:
Woops, guess I misunderstood that instruction